### ISSUE TITLE
[geometry] Matrix method inverse() doesn't suggest an inverse algorithm

### ISSUE THREAD

**Issue by dirkschulze** [w3c/fxtf-drafts#271]
Original post by Adenilson Cavalcanti on https://www.w3.org/Bugs/Public/show_bug.cgi?id=26374:

>A quick look into wikipedia lists about 8 different methods for calculating a matrix inverse.
>
>It might be appropriated to suggest which algorithm should be used to help different UAs to come to similar results.

Response from @zcorpan 
>Dirk, Rik, do you know how to test this to tell which algorithm an implementation uses?

**Comment by dirkschulze** [comment]
IMO this should be up to the implementation.

**Comment by tabatkins** [comment]
Yeah, inverting has a single *definition*; aside from some numerical stability issues (which we generally leave up to quality-of-implementation), all the methods give you the same result.
